client/src/components/reader/Header.js

```diff
@@ -157,7 +159,21 @@ export default class Header extends Component {
           visible={this.props.visibility.tocDrawer}
           hideTocDrawer={this.triggerHideToc}
         />
-        <nav className="menu-panels">
+        <nav className="menu-panels-left">
+          <UIPanel
+            id="readerReturn"
+            visibility={this.props.visibility.uiPanels}
+            bodyComponent={ReturnMenuBody}
+
+            // Props required by body component
+            projectId={this.props.text.relationships.project.id}
+            projectTitle="The Project Title"
```

Comment:
Should be accessed from `texts.relationships.project.attributes.title` rather than hard coded.